pkg/apis/sriovnetwork/v1/sriovnetwork_types.go

```diff
@@ -19,11 +19,17 @@ type SriovNetworkSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=4096
 	// VLAN ID to assign for the VF. Defaults to 0.
-	Vlan     int   `json:"vlan,omitempty"`
-	// VF spoof check
-	SpoofChk *bool `json:"spoofChk,omitempty"`
-	// VF trust mode
-	Trust    *bool `json:"trust,omitempty"`
+	Vlan		int   `json:"vlan,omitempty"`
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=7
+	// VLAN QoS ID to assign for the VF. Defaults to 0.
+	VlanQoS		int   `json:"vlanQoS,omitempty"`
+	// VF spoof check, (on|off)
```

Comment:
Please add `+kubebuilder:validation:Enum=...` for CRD validation for all the new feilds.

Reply:
fixed